src/js/jsx/tools/GuidesOverlay.jsx

```diff
@@ -233,6 +242,40 @@ define(function (require, exports, module) {
         },
 
         /**
+         * Returns true if the given D3 component is under the given mouse location
+         *
+         * @private
+         * @param {SVGElement} zone Guide zone being checked
+         * @param {number} x mouse X coordinate
+         * @param {number} y mouse Y coordinate
+         * @return {boolean} True if [x,y] is in the zone
+         */
+        zoneUnderLocation: function (zone, x, y) {
+            var zoneLeft = mathUtil.parseNumber(zone.attr("x")),
+                zoneTop = mathUtil.parseNumber(zone.attr("y")),
+                zoneRight = zoneLeft + mathUtil.parseNumber(zone.attr("width")),
+                zoneBottom = zoneTop + mathUtil.parseNumber(zone.attr("height"));
+            
+            return zoneLeft < x && zoneRight > x &&
```

Comment:
If the zone is not ready for this, it should return false, we don't want to highlight it. I don't see mathUtil.parseNumber returning null in this function given we set those attributes. I'll make the <= changes.